### ISSUE TITLE
kyvejs incorrect typings on LCD client responses

### ISSUE THREAD

**Issue by bz888** [KYVENetwork/kyvejs#131]
With queries for the lcdClient
```
await this.lcdClient.kyve.query.v1beta1.finalizedBundle({
  pool_id: this.poolId,
  id: bundleId.toString(),
});
```

https://github.com/KYVENetwork/kyvejs/blob/38ea7c72ffd0f199426e793ab93afdb9e16d5198/common/types/src/client/kyve/query/v1beta1/bundles.ts#L96

the response type is this, but my reponse i would get is of type FinalizedBundle
https://github.com/KYVENetwork/kyvejs/blob/38ea7c72ffd0f199426e793ab93afdb9e16d5198/common/types/src/client/kyve/query/v1beta1/bundles.ts#L9

potential issue with generated ts types.

**Comment by troykessler** [comment]
Thanks for submitting, we will try to get this fixed

**Comment by troykessler** [comment]
Issue is in the proto definitions for the QueryFinalizedBundle proposal: https://github.com/KYVENetwork/chain/blob/main/proto/kyve/query/v1beta1/bundles.proto#L130

Solution will be available once this has been fixed in the next v1.5 chain upgrade
